tests/tests_integration/test_api/test_relationships.py

```diff
@@ -118,11 +118,7 @@ def test_retrieve_unknown(self, new_relationship):
     def test_list_filter(self, create_multiple_relationships):
         relationships_ext_ids, ext_id, source_ext_id = create_multiple_relationships
         res = API_REL.list(source_external_ids=[source_ext_id[3]])
-        res2 = API_REL.list(confidence={"min": 1, "max": 1})
```

Comment:
This is a non deterministic test (not unique) and so when github actions runs all four versions of pytest, it picks up relationships created in other version. (Given all of them are run at the same time on same VM, similar to issue when creating something with a non-random external id).